torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8780,6 +8780,15 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
         supports_autograd=True,
         supports_forward_ad=True,
     ),
+
+    ForeachFuncInfo(
+        'sign',
+        dtypes=floating_types_and(torch.bool, torch.bfloat16, torch.half),
```

Comment:
oh interesting that bool is also supported :o

Reply:
I copy paste it from `torch.sign`'s opinfo, IIUC

Reply:
Yea it makes sense, I was pleasantly intrigued.